src/SignalR/server/Specification.Tests/src/Microsoft.AspNetCore.SignalR.Specification.Tests.csproj

```diff
@@ -20,6 +20,7 @@
     <Reference Include="Microsoft.AspNetCore.SignalR.Protocols.Json" />
     <Reference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" />
     <Reference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" />
+    <Reference Include="Microsoft.AspNetCore.Testing" />
```

Comment:
This one's a problem, Microsoft.AspNetCore.SignalR.Specification.Tests is actually a shipping package, not a test. It can't depend on Microsoft.AspNetCore.Testing which is not a shipping package.
https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR.Specification.Tests

Reply:
Right, this one can't use any test helpers.

Reply:
I will double check this one.

Reply:
Both ```HubLifetimeManagerTestBase``` and ```ScaleoutHubLifetimeManagerTests``` uses DefaultTimeout heavily. Which makes me move TaskExtensions from Microsoft.AspNetCore.Testing to \Shared folder to reference the file directly to resolve this issue.

Reply:
This gets a little more tricky in TestClient.cs to make it compile everywhere, I have to use compiler conditions in the using statements.